sdk/communication/Azure.Communication.Administration/tests/samples/Sample2_PhoneNumberAdministrationClient.cs

```diff
@@ -165,7 +173,8 @@ public void PurchaseAndRelease()
 
             while (!reserveOperation.HasCompleted)
             {
-                Thread.Sleep(2000);
+                //@@ Thread.Sleep(2000);
```

Comment:
@pavelprystinka, similarly I added this: https://github.com/Azure/azure-sdk-for-net/pull/18037/files#diff-5636043a2134cb7dcd123d58c6625cf1dd400c6daa1ad7879fe08e6a97e8be2cR245 to only sleep in record and live mode